tests/scripts/thread-cert/node.py

```diff
@@ -4052,7 +4063,14 @@ def add_ipmaddr_tun(self, ip: str):
         cmd = f'python3 /app/third_party/openthread/repo/tests/scripts/thread-cert/mcast6.py {self.TUN_DEV} {ip} &'
         self.bash(cmd)
 
-    def get_ip6_address(self, address_type: config.ADDRESS_TYPE):
+    def get_ip6_address(self, address_type: config.ADDRESS_TYPE, interface_type=None):
+        if interface_type == config.INTERFACE_TYPE.INFRA_ETH:
+            return super().get_ip6_address(address_type)
+        elif interface_type == config.INTERFACE_TYPE.THREAD_WPAN:
+            return super(LinuxHost, self).get_ip6_address(address_type)
```

Comment:
The above usage is: 
`br2_thread_link_local = br2.get_ip6_address(config.ADDRESS_TYPE.LINK_LOCAL, config.INTERFACE_TYPE.THREAD_WPAN)`

Then it will goes to:
`return super(LinuxHost, self).get_ip6_address(address_type)`

Next in `LinuxHost::get_ip6_address`:
```
        if address_type == config.ADDRESS_TYPE.LINK_LOCAL:
            return self._getInfraLinkLocalAddress()
```

Then it returns the infra link local address, not the Thread link local address. This seems not correct?

Reply:
Thanks!

`super(LinuxHost, self).get_ip6_address(address_type)` actually calls `NodeImpl.get_ip6_address(address_type)` which is expected